app/src/main/java/me/ccrama/redditslide/Activities/FullscreenImage.java

```diff
@@ -36,7 +36,7 @@
 /**
  * Created by ccrama on 3/5/2015.
  */
-public class FullscreenImage extends BaseActivity {
+public class FullscreenImage extends FullScreenActivity {

```

Comment:
Why extending it from a new class when you already checked for it in line 51-54?